src/main/java/control/ProjectController.java

```diff
@@ -189,13 +233,67 @@ private void addCameraShotForLoad(int cameraIndex, CameraShot shot) {
     }
     
     /**
+     * Start the edit project modal with information of the current project filled in.
+     */
+    public void editProject() {
+        editProjectModal = new EditProjectModalView(controllerManager.getRootPane(), true);
+        editProjectModal.getAddCameraButton().setOnMouseClicked(this::addCamera);
+        editProjectModal.getDeleteCameraButton().setOnMouseClicked(this::deleteCamera);
+        editProjectModal.getAddCameraTypeButton().setOnMouseClicked(this::addCameraType);
+        editProjectModal.getDeleteCameraTypeButton().setOnMouseClicked(this::deleteCameraType);
+        editProjectModal.getCancelButton().setOnMouseClicked(this::cancel);
+        editProjectModal.getCreationButton().setOnMouseClicked(this::save);
+    }
+    
+    /**
+     * Handler for deleting a camera type.
+     * @param event the MouseEvent for this handler
+     */
+    private void deleteCameraType(MouseEvent event) {
+        int selectedIndex = editProjectModal.getCameraTypeList()
+                                            .getSelectionModel().getSelectedIndex();
+        if (selectedIndex != -1) {
+            editProjectModal.getCameraTypes().remove(selectedIndex);
+            editProjectModal.getCameraTypeList().getItems().remove(selectedIndex);
+        } else {
+            log.debug("TODO: Error message select camera");
+            // TODO: Error message, select camera type
```

Comment:
You can just follow style for this in most modals: replace the current titleLabel text with an error message, and set titleLabel color to red. For later, ofc.